tests/server/cluster/cluster_test.go

```diff
@@ -565,6 +566,7 @@ func TestStoreVersionChange(t *testing.T) {
 
 func TestConcurrentHandleRegion(t *testing.T) {
 	re := require.New(t)
+	deadlock.Opts.DeadlockTimeout = time.Minute
```

Comment:
The default value is 30s, which will report potential deadlock when running the concurrent region heartbeat test since the deadlock detection will cost much time.